Anatoly Shlom/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Paris</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css">
+</head>
+<body>
+    <main>
+        <article class="wdata">
+            <section class="w_date">
+                <div class="cur_week_day">
+                    Friday
+                </div>
+                <div class="cur_date">
+                    August, 23rd
+                </div>
+            </section>
+            <section class="info">
+                <div class="temp">
+                    12&#176;
+                </div> 
+                <div class="loc">
+                    Paris, France
+                </div>
+                <div class="wind">
+                    <div class="wind_img">
+                        <img src="img/Wind.png" alt="Wind">
+                    </div>
+                    <div class="wind_speed">    
+                        12
+                    </div>
+                    <div class="speed_format">
+                        mph
+                    </div>
+                </div>
+            </section>
+        </article>
+        <nav class="days_nav">
+            <ul>
+                <li>
+                    <img src="img/back.png" alt="Go back">
+                </li>
+                <li>
+                    <div>
+                        <p>Monday</p>
```

Comment:
Why it's paragraph and not just a `span`?

Reply:
Fixed
But i still don't understand why it is better to use spans instead of `<div>` or for example `<p> `. Is it somehow connected to the DOM tree and influence on it?

Reply:
<p> is more semantic tag. Actually for paragraph